src/model/UserphraseModel.cpp

```diff
@@ -106,14 +107,15 @@ void UserphraseModel::remove(QModelIndexList indexList)
             userphrase_[index].bopomofo_.toUtf8().constData());
         if (ret > 0) {
             removerecord_.push_back(Userphrase{userphrase_[index]});
-            // FIXME: std::vector::erase is an inefficient operation.
+            remove_cnt++;
+			// FIXME: std::vector::erase is an inefficient operation.
             userphrase_.erase(userphrase_.begin() + index);
         } else {
             qWarning() << "chewing_userphrase_remove() returns" << ret;
         }
         // FIXME: Handle chewing_userphrase_remove fails.
     }
-
+	maxundocnt = remove_cnt;
```

Comment:
I use vector to store in [new commit](https://github.com/chewing/chewing-editor/pull/233/commits/98f60c216e61464be87d5574f57bdb12a1b2f69d)
Is there any question?